plugins/flutter_aepcore/android/src/main/java/com/adobe/marketing/mobile/flutter/flutter_aepcore/FlutterAEPCorePlugin.java

```diff
@@ -37,6 +41,10 @@ public class FlutterAEPCorePlugin implements FlutterPlugin, MethodCallHandler {
     public void onAttachedToEngine(@NonNull final FlutterPluginBinding binding) {
         channel = new MethodChannel(binding.getBinaryMessenger(), "flutter_aepcore");
         channel.setMethodCallHandler(new FlutterAEPCorePlugin());
+        Context appContext = binding.getApplicationContext();
+        if (appContext instanceof Application) {
+            application = (Application) appContext;
+        }
```

Comment:
Can you also add a call to `MobileCore.setWrapperType(WrapperType.Flutter)` here?